.changeset/wild-doodles-shop.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-techdocs-node': major
+---
+
+There was an issue in the uploading of large size files to the AWS S3. We have modified the logic by adding retry along with multipart uploading functionality.
```

Comment:
I don't see any changes for what's described here in this PR, did you miss committing some changes?